src/styles/blocks/card.scss

```diff
@@ -0,0 +1,74 @@
+.card {
+  width: 200px;
+  border: 1px solid $light-color-border;
+  border-radius: 5px;
+  background-color: $white-color-link;
+  padding: 32px 16px 16px;
+
+  &__photo {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__header {
+    margin: 0 0 4px;
```

Comment:
Would be better to reset margins in separate scss file instead of reset reset the property for each element